### ISSUE TITLE
no battery status

### ISSUE THREAD

**Issue by alexandrekysli** [KrolSeb/HP-Pavilion-Gaming-15-dk00xxxx-Hackintosh-OpenCore#3]
Hello guy and thank for all your help.
I have successfully install Big Sur on my hp pavilion 15 but after debugging some error (graphic, sound and usb), now i have not battery status.
I would like know if you have use "How to patch DSDT" from rehabman to pass this step or you don't have save problem this your experience. Thank for all and sorry for my bad english, i speak french.

**Comment by akardas16** [comment]
> Hello guy and thank for all your help.
> I have successfully install Big Sur on my hp pavilion 15 but after debugging some error (graphic, sound and usb), now i have not battery status.
> I would like know if you have use "How to patch DSDT" from rehabman to pass this step or you don't have save problem this your experience. Thank for all and sorry for my bad english, i speak french.

Hi, how did you deal with ACPI errors? I stuck with these errors.

**Comment by alexandrekysli** [comment]
Can you send me a screenshot of this error

**Comment by akardas16** [comment]
> Can you send me a screenshot of this error

![1626031829206](https://user-images.githubusercontent.com/28716129/129875737-65096d20-b1f3-4ecd-82fb-4e57ee633a30.jpg)

**Comment by akardas16** [comment]
> Can you send me a screenshot of this error

I tried many times. The error is persistent

**Comment by alexandrekysli** [comment]
ok
What is your version of hp pavilion ?
For mine, it's pavilion 15-cs3153cl come with IceLake Core i5 (10th gen).
I speak about this because in my ACPI folder in EFI/OC, i use only this .aml:
[SSDT-XOSI] - [SSDT-AWAK] - [SSDT-PNLF-CFL] - [SSDT-USBX] - [SSDT-EC] - [SSDT-PLUG]

So give me your pavilion 15 model name and your EFI folder for check it 😎

**Comment by akardas16** [comment]
the model is HP Pavilion Gaming 15-DK1008NT(Intel Core i5 10300H 8GB 512GB SSD 1650Ti Freedos 15.6" FHD). I tried different EFI s. here is my last EFI: https://drive.google.com/file/d/1mv0f0o_aKub7hd1g_GCHfOLN_0xYrmqB/view?usp=sharing
Also I added to report of my device
[report[2305843009214570069].txt](https://github.com/KrolSeb/HP-Pavilion-Gaming-15-dk00xxxx-Hackintosh-OpenCore/files/7006791/report.2305843009214570069.txt)

**Comment by alexandrekysli** [comment]
Your laptop has some simillars item with mine... So i give you my efi file but key in mind that you have to edit config.plist for great support of your laptop. Also please choose the catalina recovery image for better debugging with booting. i did it for mine.

My EFI:
https://drive.google.com/file/d/1oIS2t_dv8d46b3qhSEqVDGHCkE7vopsC/view?usp=sharing

PS: Remove [AirportItlwm.kext] [IntelBluetoothFirmware.kext] [IntelBluetoothInjector.kext] because your have not Intel Wifi and Bluetooth. After that rebuild config.plist with Ctrl+R in ProperTree

Catalina recovery from dortania guide with macrecovery.py in OpenCorePkg :
`python macrecovery.py -b Mac-00BE6ED71E35EB86 -m 00000000000000000 download`
PS: If you're enabled to pass kernel load and boot installer you can replace this image by the big sur recovery

**Comment by akardas16** [comment]
> PS: Remove [AirportItlwm.kext] [IntelBluetoothFirmware.kext] [IntelBluetoothInjector.kext] because your have not Intel Wifi and Bluetooth. After that rebuild config.plist with Ctrl+R in ProperTree

I just did that changes to your EFI but same error occured.

**Comment by alexandrekysli** [comment]
Same error screen ?
Did you make the BIOS settings and use Catalina recovery image ?

**Comment by akardas16** [comment]
> Same error screen ?
> Did you make the BIOS settings and use Catalina recovery image ?

yes same error. I didn't use Catalina but I will try when I have enough time to download Catalina iso. I think Bıos settings are correct. What should I change?

**Comment by alexandrekysli** [comment]
In fact when i use catalina dmg, i am able to see more debugging error info.
With, i can have more info for debugging that.
Use catalina image guy. It's just for testing kernel booting

**Comment by sirepson** [comment]
Support macOS Monterey?
